docs/src/doc/user-guide/exporting.md

```diff
@@ -65,3 +65,16 @@ On android, we do not embed a JVM, we use the existing ART provided by the OS. I
 
 !!! warning
     Similar to the desktop targets, the game copies the needed jar files to the `user://` directory upon first execution or if the files have changed. On android this is the applications `files` folder. If you do IO operations on Android, never empty the whole `files` folder! Only delete what you have added or exclude the following two files when clearing the `files` folder: `godot-bootstrap-dex.jar` and `main-dex.jar`.
+
+## GraalVM Native Image
+
+!!! warning
+    GraalVM native image is a advanced feature and requires a lot of work to support. Especially if you rely on many third party libraries.
+
+In order to build for graalvm, follow `GraalVM native-image` section in advanced user guide.
```

Comment:
One tiny thing: I would link the corresponding page here: `GraalVM native image`